File: Case.java
public class Case {
    public static void main(String[] args) {
        String phrase = "how cAn mirror BE";
        System.out.println(toCase(phrase));
    }

    public static String toCase(String phrase) {
        char[] charArray = phrase.toCharArray();

        for (int i = 0; i < charArray.length; i++) {
            if (i == 0 || charArray[i - 1] == ' ') {
                charArray[i] = Character.toUpperCase(charArray[i]);
            } else charArray[i] = Character.toLowerCase(charArray[i]);
        }
        return new String(charArray);
    }
}

File: case.py
def case(s):
    return " ".join(w.capitalize() for w in s.split())


def case_string(s):
    new_s= " "
    
    for index, i in enumerate(s):
        if index == 0 or s[index-1] == " ":
            new_s += s[index].upper()
        else:
            new_s += i.lower()
    return new_s

n = "how cAn mirror BE"
print(case_string(n))